src/Behavioral.Automation/Bindings/DropdownBinding.cs

```diff
@@ -116,7 +116,7 @@ public void CheckMultipleSelectedValues([NotNull] IMultiSelectDropdownWrapper wr
         {
             foreach (var row in table.Rows)
             {
-                runnerAction($"the \"{row.Values.First()} \" value {behavior} {enabled} in {wrapper.Caption}:");
+                runnerAction($"the \"{row.Values.First()}\" value {behavior} {enabled} in {wrapper.Caption}");
```

Comment:
Are we sure that row.Values are not empty?

Reply:
Not really. Added assert to check it